R/httr.R

```diff
@@ -1,14 +1,10 @@
 httr_get <- function(url) {
   pieces <- httr::parse_url(url)
-
-  # cat("Url parts: \n")
-  # utils::str(list(
-  #   url = url,
-  #   pieces = pieces
-  # ))
-
-  if (!pingr::is_up(pieces$hostname, pieces$port)) {
-    abort("Shiny app is no longer running")
+  # Only check if no basic auth is being used
```

Comment:
@schloerke I wonder why this condition is necessary. `pingr::is_up` does not know anything about HTTP auth (right?), so it should be able to contact a server with HTTP auth just as well as one without it. Even more, it is possible that a certain path on that HTTP server is only accessible with HTTP auth, but the root `/` is freely accessible - one example being https://httpbin.org/basic-auth/user/passwd and https://httpbin.org/ - in which case even if `pingr::is_up` did know something about HTTP auth, it would be able to check the root given just `hostname` and `port`. So I'd  think `if (is.null(pieces$username) && is.null(pieces$password))` can be removed completely.

Reply:
Perfect. Thank you!